libexec/scoop-import.ps1

```diff
@@ -0,0 +1,176 @@
+# Usage: scoop import <filepath/filepaths> [options]
+# Summary: Import exported list!
+# Help: The basic way to install exported list:
+#      scoop import exportedfile.txt
+# It also supports for multiple file:
+#      scoop import file1.txt file2.txt
+
+function createAppsList($filePaths) {
+    $appList = New-Object System.Collections.ArrayList
+    if ($filePaths) {
+        foreach ($file in $filePaths) {
+            if (checkFileExt($file)) {
+                foreach ($line in (Get-Content $file | Select-Object -Unique)) {
+                    if (checkEmptyLine($line)) {
+                        if (checkComment($line)) {
+                            continue
+                        }
+                        else {
+                            $appList += ($line + "`r`n")
+                        }
+                    }
+                }
+            }
+            else {
+                $extError = "Please select .txt file!"
+                throw $extError
+            }
+        }
+    }
+    else {
+        $filePathsError = "There is no file to exported!"
+        throw $filePathsError
+    }
+
+    $uniqueAppList = $appList | Get-Unique
+
+    return $uniqueAppList
+}
+
+function checkFileExt ($file) {
+    $ext = [IO.Path]::GetExtension($file)
+    if ($ext -eq ".txt") {
+        return $True
+    }
+    else {
+        return $false
+    }
+}
```

Comment:
I believe above can be reduced to this.

```
function checkFileExt ($file) {
    return ([IO.Path]::GetExtension($file) -eq ".txt")
}
```

The above simplification can be applied to functions below and would make the code more concise and thus make it more readable.